resolwe/flow/tests/test_collections.py

```diff
@@ -0,0 +1,93 @@
+# pylint: disable=missing-docstring
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import AnonymousUser
+from django.urls import reverse
+
+from guardian.shortcuts import assign_perm
+from rest_framework.test import APITestCase
+
+from resolwe.flow.models import Collection, Data, Process
+
+
+class CollectionTests(APITestCase):
```

Comment:
There is already a [TestCollectionViewSetCase](https://github.com/genialis/resolwe/blob/b52d1223a98efe676758e6b65c4dd691696edb3d/resolwe/flow/tests/test_api.py#L431), so it would make sense to merge them.